client/components/utils/getProductQueries.js

```diff
@@ -0,0 +1,10 @@
+export default (search) => {
+  return search
+    .slice(1)
+    .split('&')
+    .reduce((acc, param) => {
+      const [key, value] = param.split('=');
+      acc[key] = value;
+      return acc;
+    }, {});
+};
```

Comment:
This is a utility function that turns a `search` string from React Router into an object, so we can grab specific values and update state